File: android/src/main/java/com/zoontek/rnbootsplash/RNBootSplash.java
package com.zoontek.rnbootsplash;

import android.app.Activity;

import androidx.annotation.NonNull;
import androidx.annotation.StyleRes;

public class RNBootSplash {

  public static void init(@NonNull final Activity activity, @StyleRes int themeResId) {
    RNBootSplashModuleImpl.init(activity, themeResId);
  }
}

File: android/src/main/java/com/zoontek/rnbootsplash/RNBootSplash.kt
package com.zoontek.rnbootsplash

import android.app.Activity

import androidx.annotation.StyleRes

object RNBootSplash {

  fun init(activity: Activity, @StyleRes themeResId: Int) {
    RNBootSplashModuleImpl.init(activity, themeResId)
  }
}
